airflow/gcp/operators/pubsub.py

```diff
@@ -326,34 +505,68 @@ class PubSubSubscriptionDeleteOperator(BaseOperator):
         For this to work, the service account making the request
         must have domain-wide delegation enabled.
     :type delegate_to: str
+    :param retry: (Optional) A retry object used to retry requests.
+        If None is specified, requests will not be retried.
+    :type retry: google.api_core.retry.Retry
+    :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+        to complete. Note that if retry is specified, the timeout applies to each
+        individual attempt.
+    :type timeout: float
+    :param metadata: (Optional) Additional metadata that is provided to the method.
+    :type metadata: Sequence[Tuple[str, str]]]
+    :param project: (Deprecated) the GCP project ID where the topic will be created
+    :type project: str
     """
-    template_fields = ['project', 'subscription']
+    template_fields = ['project_id', 'subscription']
```

Comment:
```suggestion
    template_fields = ['project', 'project_id', 'subscription']
```

Reply:
I had to revert these changes (only `"project"` shall remain) because of the errors due to errors:
`  File "/workspace/airflow/models/baseoperator.py", line 662, in _do_render_template_fields
    content = getattr(parent, attr_name)
AttributeError: 'PubSubPublishOperator' object has no attribute 'project'`

Reply:
Should be OK. Maybe (nice to have) we could add a warning in deprecation message, that templates won't work ?